.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,23 @@
+<!-- Hey there. Thank you so much for improving NewPipe. Please take a moment to fill out the following suggestion on how to structure this PR description. Having roughly the same layout drastically helps everybody :)-->
+
+#### What is it?
+- [ ] Bug fix
+- [ ] Feature
+
+#### Long description of the changes in your PR
+*Now you can make videos*
+
+#### Fixes the following issue(s)
+<!--Also add reddit or other links where you change is related-->
+- 
+
+#### Relies on the following changes
+<!-- Delete this if it doesn't apply to you -->
+- 
+
+#### Testing apk
+<!--Ensure to have your changes on a new branch which has a meaningful name. This name will be used as suffix of the app id to allow installing and testing multiple versions of NewPipe. Do NOT name your branches like "patch-0" and "feature-1".-->
```

Comment:
Ensure that you have

as a suffix

For example, if your PR implements a bug fix for comments, a good branch name could be "commentfix"